src/style.css

```diff
@@ -0,0 +1,59 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+.search-input {
+  height: 70px;
+  width: 100%;
+  font-family: "Avenir", sans-serif;
```

Comment:
usually it is defined in html selector

Reply:
![image](https://user-images.githubusercontent.com/127408751/225360977-bfa5b9a8-fa80-416d-9f6f-17e6b5f0e819.png)

yes but I should then skip the default values ​​of the browser in *

Reply:
> usually it is defined in html selector


maybe i don't understand something